lms/static/scripts/frontend_apps/types.d.ts

```diff
@@ -1,3 +1,5 @@
+/// <reference types="navigation-api-types" />
```

Comment:
It seems that `import "navigation-api-types"` also works to import these types. I thought `<reference ...>` comments were mostly a legacy feature, but I don't know what the preferred approach is to import types from packages in the latest TS version.